src/Conversion/InterventionImageFileConverter.php

```diff
@@ -66,7 +69,7 @@ function (AssetStore $store, string $filename, string $hash, string $variant) us
                     return [$tuple, $backend];
                 }
             );
-        } catch (ImageException $e) {
+        } catch (RuntimeException $e) {
```

Comment:
Note that this is Intervention's special exception, not the base one.... I can add an alias if you want to avoid confusion.